icons/generate.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash -xe
+
+src=../logomark/for-dark-background.svg
+
+function gen() {
+  inkscape -z -w $1 -h $1 -y 0.0 $src -o $2
```

Comment:
Can't we use imagemagick here? It hurts me a lot to say that, by inkscape sucks on Mac.

Reply:
you mean mac sucks on inkscape?

according to my quick google research inkspace is better for working on svgs, imagemagick has some svg support but it's not as good as it is not the focus. if I needed to process the png after I would definitely use imagemagick then